### ISSUE TITLE
Add a lang attribute to the switch language button

### ISSUE THREAD

**Issue by maxneuvians** [veteransaffairscanada/vac-benefits-directory#978]
See #380

**Comment by smcmurtry** [comment]
Here is a guideline: https://www.w3.org/International/questions/qa-html-language-declarations

TLDR: The button text "Français" is in french so it should have a `lang="fr"`. The button could be wrapped in a `<span>` with `title="French"` as helper text for english speakers.
